test/helpers/deployZNS.ts

```diff
@@ -207,7 +243,7 @@ export const deployZNS = async ({
     deployer,
     accessController.address,
     priceOracle.address,
-    zeroTokenMock.address,
```

Comment:
it's still a mock. not actual zero token. let's specify this, so nobody deploys it by mistake to a network

Reply:
Makes sense, I'll add this as well as a comment specifying that this script is used for testing only